gremlin-go/driver/graph.go

```diff
@@ -97,14 +98,15 @@ func (p *Path) String() string {
 	return fmt.Sprintf("path[%s]", strings.Trim(strings.Join(strings.Fields(fmt.Sprint(p.objects)), ", "), "[]"))
 }
 
-func (p *Path) GetPathObject(key string) (error, interface{}) {
+// GetPathObject returns the value that corresponds to the key for the Path.
+func (p *Path) GetPathObject(key string) (interface{}, error) {
 	if len(p.objects) != len(p.labels) {
-		return errors.New("Path is invalid because it does not contain an equal number of labels and objects."), nil
+		return nil, errors.New("Path is invalid because it does not contain an equal number of labels and objects.")
```

Comment:
Should we use a localization resource for messages?

Reply:
I don't think our localization works with errors as of right now @simonz-bq ?

Reply:
This is correct.